sdk/python/feast/feature_store.py

```diff
@@ -1895,6 +1905,259 @@ def _retrieve_from_online_store(
             )
         return read_row_protos
 
+    @staticmethod
```

Comment:
@tsisodia10 I think this is some sort of merge issue, right? I moved all these static methods to utils in another PR.